au4/src/au3wrap/internal/processinginteraction.h

```diff
@@ -16,6 +16,7 @@ class ProcessingInteraction : public processing::IProcessingInteraction
 
     bool changeClipStartTime(const processing::ClipKey& clipKey, double sec) override;
     bool changeClipTitle(const processing::ClipKey& clipKey, const muse::String& newTitle) override;
+    bool removeClip(const processing::ClipKey& clipKey, double begin, double end) override;
```

Comment:
Non-obvious signature. I think it's better to have two methods, something like:
* bool removeClip(const processing::ClipKey& clipKey);
* bool removeClipData(const processing::ClipKey& clipKey, double begin, double end);